renderer/src/GLRenderer.cpp

```diff
@@ -407,7 +412,7 @@ void GLRendererImplementation::renderFrame() {
         meshes.reserve(objects_.size());
 
         for (const auto &[id, object] : objects_) {
-            auto mesh_component = object.getComponent("mesh");
+            auto mesh_component = object.getComponent<geometry::Mesh>();
```

Comment:
`mesh_component_ptr`

Reply:
Left this one as is. Removed the above one, for consistency.